src/SprykerEco/Client/Computop/ComputopClientInterface.php

```diff
@@ -77,7 +89,8 @@ public function savePaydirektInitResponse(QuoteTransfer $quoteTransfer);
     public function saveCreditCardInitResponse(QuoteTransfer $quoteTransfer);
 
     /**
-     * {@inheritDoc}
+     * Specification:
+     * - Saves PayNow response to DB
```

Comment:
* Saves PayNow response to the database.

Reply:
++